tests/devlxd-vm

```diff
@@ -90,14 +95,47 @@ fi
 # Wait for boot to be completed to interact with snap/snapd
 waitInstanceBooted v1
 
+# Below tests for image export rely on an event being emitted on export. This will not be backported to 4.0 or 5.0,
+# so exit the test here.
+if [ "${LXD_SNAP_CHANNEL}" = "4.0/edge" ] || [ "${LXD_SNAP_CHANNEL}" = "5.0/edge" ]; then
```

Comment:
Could we bail out if `devlxd_images_vm` (when it's a thing) is not detected?

Reply:
I've just removed this block so that the test continues. This way we're still testing that the nested VM works on older versions. I'm going to revisit the container tests to do the same thing.

Reply:
Maybe, just remove the `event_lifecycle` to avoid running into a merge conflict.

Reply:
Done.